### ISSUE TITLE
MySQL: change Warning: #1287 change GRANT to CREATE USER (deprecated)

### ISSUE THREAD

**Issue by winkler-winsen** [csev/wa4e#24]
MSQL informs:

> Warning: #1287 Using GRANT for creating new user is deprecated and will be removed in future release. Create new user with CREATE USER statement.

readme.md
```mysql
CREATE DATABASE tsugi DEFAULT CHARACTER SET utf8;
GRANT ALL ON tsugi.* TO 'ltiuser'@'localhost' IDENTIFIED BY 'ltipassword';
GRANT ALL ON tsugi.* TO 'ltiuser'@'127.0.0.1' IDENTIFIED BY 'ltipassword';
```

https://github.com/csev/wa4e/blame/7bb454d89bec060f482dc2bdd2cb27a09bc265b9/README.md#L26
https://github.com/csev/wa4e/blame/7bb454d89bec060f482dc2bdd2cb27a09bc265b9/README.md#L27

Using MAMP 4.2.0.23973
- MySQL 5.7.24

**Comment by csev** [comment]
Thanks.

**Comment by csev** [comment]
Fixed in #6a5fe20b1693c17d9b186ed12ac34d7ba1383a44
